### ISSUE TITLE
Cannot set fonts

### ISSUE THREAD

**Issue by KarolS** [garglk/garglk#351]
Using the latest 2019.1 release on Windows 10.

I put the following in garglk.ini:

```
monofont      Fantasque Sans Mono
monosize      16

propfont      Arial
propsize      16
```

Here's what I got:

![2019-11-03 22_10_00-Laid Off from the Synesthesia Factory - Katherine Morayati](https://user-images.githubusercontent.com/1790750/68091981-b6f39400-fe86-11e9-927f-2ab3cc8f1ec9.png)

As you can see, neither font is correct and I can't change that.

I also tweaked other settings and they seemed to work, so the config file is being picked up.

**Comment by cspiegel** [comment]
I don't have Windows installed so I can't test directly.  Fonts are working for me using Wine on Linux, however, so I can't reproduce this.  @mbikovitsky, you did some Windows font work recently.  If you have some time, would you be able to try to reproduce this?

**Comment by mbikovitsky** [comment]
Sure, I should have some time on the weekend to look into this.

@KarolS, is [this](https://github.com/belluzj/fantasque-sans) the same Fantasque font you were using?

**Comment by KarolS** [comment]
Yes, that's the one.

For reference, this old build works for me perfectly: https://github.com/garglk/garglk/issues/285#issuecomment-341520320

**Comment by mbikovitsky** [comment]
Right, so I think I found the issue. @KarolS, does this seem about right?

Before:
![image](https://user-images.githubusercontent.com/1389811/68957578-fa2cfb80-07d2-11ea-8c67-aea9ee33fb8d.png)

After:
![image](https://user-images.githubusercontent.com/1389811/68957638-1d57ab00-07d3-11ea-8752-097a2c19b9e5.png)

**Comment by KarolS** [comment]
The screenshots look correct. 
I can't build the source, so I can't test it myself.

**Comment by cspiegel** [comment]
I've released 2019.1.1 with this fix included.

**Comment by KarolS** [comment]
I tested the release and it's fixed, I can finally set my favourite fonts again!

Thanks!
